neo4j/neo4j-bundle/0.3/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+    "bundles": {
+        "Neo4j\\Neo4jBundle\\": ["all"]
```

Comment:
is namespace correct here? Probably it should be `Neo4j\\Neo4jBundle\\Neo4jBundle`

Reply:
It is :)